source/03_development/04_docker_images/01_ehrbase/02_use_image/002_using_docker_image.rst

```diff
@@ -0,0 +1,85 @@
+Run EHRbase in Docker
+=====================
+
+.. note:: 
+
+    Remember: EHRbase requires a properly configured and running PostgreSQL DB to work.
+    Make sure to set this up first before you try run EHRbase.
+
+To run EHRbase in a Docker Container first pull the official Docker image from Docker Hub:
+
+.. code-block:: bash
+
+    docker pull ehrbaseorg/ehrbase
+
+
+**OR**
+ 
+build your own image form Dockerfile:
+
+
+.. code-block:: bash
+
+    git clone https://github.com/ehrbase/ehrbase.git
+    cd ehrbase
+    docker build -t myehrbase/ehrbase .
+    docker image ls
+
+
+**THEN** use the `docker run` command adjusting parameters to your needs to change Container's default behaviour.
+
+.. note:: 
+
+    Remember: Container's default behaviour is set during Docker image build time.
+
+
```

Comment:
These are the correct names for the variables (starting after SECURITY_AUTHTYPE)

| Parameter | Usage | Example |
| ----------- | ------- | --------- |
| SECURITY_AUTHUSER | Username for BASIC auth | myuser |
| SECURITY_AUTHPASSWORD | Password for BASIC auth user | myAwesomePassword123 |
| SECURITY_AUTHADMINUSER | Username for BASIC auth admin | myadmin |
| SECURITY_AUTHADMINPASSWORD | Password for BASIC auth admin | mySuperAwesomePassword123 |
| SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI | OAuth2 server isuer uri | https://keycloak.example.com/auth/realms/ehrbase |
| ADMINAPI_ACTIVE | Should admin endpoints be enabled | true |
| ADMINAPI_ALLOWDELETEALL | Is it allowed to delete all resources over admin endoint, e.g. all EHRs | true |

These values should also be added to the Dockerfile ARG and ENV values

Reply:
@axelsiebert  I'm not sure if it is a good idea to add this stuff with default values to Docker file. I'd like to pass this as a question to @StefanWinterfeldtVitasystems

Reply:
I created an issue for that https://github.com/ehrbase/ehrbase/issues/318